Fix bugs and mobile view in feedback page

samagragupta opened the pull request:
Fixes #2324 

Changes: 
- Fix mobile view
- Remove unnecessary close button
- Author name should be shown after loading 

Demo Link: https://pr-2325-fossasia-susi-web-chat.surge.sh

harsh-jindal commented:
@samagragupta Fixed in #2323 .Close this PR

samagragupta commented:
@harsh-jindal That PR still does not solve the mobile view problem.
![Capture](https://user-images.githubusercontent.com/32234926/59738308-69331200-927e-11e9-9d42-f87df6cfb263.JPG)
![skip](https://user-images.githubusercontent.com/32234926/59738534-376e7b00-927f-11e9-83fe-7d38003ca357.gif)

harsh-jindal commented:
@samagragupta Mobile view looks good. Resolve the conflicts.

samagragupta commented:
@harsh-jindal Resolved the conflicts

samagragupta commented:
@fragm3 Sorry for the mistake.
Made the changes

samagragupta commented:
@fragm3 Made consistent with the old PR.

samagragupta commented:
@fragm3 Now its opening.

samagragupta commented:
@akshatnitd 
![Capture](https://user-images.githubusercontent.com/32234926/59915847-371adf00-943b-11e9-9550-74b065554148.JPG)

akshatnitd commented:
Reduce the size of buttons instead in mobile view. Keep them on same line.

samagragupta commented:
@akshatnitd Done

fragm3 commented on src/components/cms/AuthorSkills/AuthorSkills.js:
```diff
@@ -144,32 +143,35 @@ class AuthorSkills extends Component {
     return (
       <React.Fragment>
-        <DialogTitle>
-          <CloseButton onClick={actions.closeModal} />
```
We need the Close Button

samagragupta replied:
Done

samagragupta commented:
@fragm3 Made the changes

fragm3 commented:
@akshatnitd do we require the author name after loading?

samagragupta commented:
@fragm3 Removed that space

samagragupta commented:
@fragm3 I think the author's name should be loaded.
@akshatnitd @harsh-jindal What are your views?

samagragupta commented:
@fragm3 And I think the mobile view is good in this PR. If you get any other issue in this PR, then please mention.

samagragupta commented:
@akshatnitd I have not created the CSS file, it was the previous PR which deleted the CSS file.
Now I have removed the file. Also removed the author's name.

harsh-jindal commented on src/components/cms/AuthorSkills/AuthorSkills.js:
```diff
@@ -159,13 +147,9 @@ class AuthorSkills extends Component {
       <React.Fragment>
-        <DialogTitle>
+        <AdminTitle>
           <CloseButton onClick={actions.closeModal} />
```
No need to keep CloseButton inside Admin Title.

harsh-jindal commented on src/components/cms/SkillFeedbackPage/SkillFeedbackPage.js:
```diff
@@ -93,6 +93,12 @@ const Pagination = styled.ul`
   align-items: center;
 `;
 
+const PaperContainer = styled(Paper)`
```
Not required. Don't make the padding 0 in mobile view. When you go to write a skill feedback you can see textfield touching the card border.

samagragupta commented:
@harsh-jindal Made the changes.
Please review.

akshatnitd commented on src/components/cms/AuthorSkills/AuthorSkills.js:
```diff
@@ -159,13 +142,7 @@ class AuthorSkills extends Component {
-          <CloseButton onClick={actions.closeModal} />
-          Skills by {author}{' '}
-          <a href={authorUrl && authorUrl !== '<author_url>' ? authorUrl : '/'}>
-            <GithubAvatarImage alt={'GitHub'} src={githubAvatarSrc} />
```
Why was the author name/url section removed? Add it.

samagragupta replied:
In this he said author name should not be loaded : https://github.com/fossasia/susi.ai/pull/2325#pullrequestreview-253098948

fragm3 commented:
@samagragupta The review above meant Author name need not be loaded after `Circular Loader`

samagragupta commented:
@akshatnitd @fragm3 Made the changes

samagragupta commented:
@fragm3 Why it is wrong the author name is loaded along with Circular Loader and after that, it is not loaded.

harsh-jindal commented:
@samagragupta Still dont see skills by author heading when i click on author name.
![Screenshot 2019-06-27 at 8 12 02 AM](https://user-images.githubusercontent.com/31013104/60229412-6d30e680-98b3-11e9-8da5-cb6535a22149.png)

samagragupta commented:
@harsh-jindal I have removed author's name by circular loader.
https://github.com/fossasia/susi.ai/pull/2325#issuecomment-505478254

samagragupta commented:
@fragm3 Made changes the same as the current version.

samagragupta commented:
@akshatnitd @harsh-jindal Please review.

fragm3 commented on src/components/cms/SkillFeedbackPage/SkillFeedbackPage.js:
```diff
@@ -83,6 +86,11 @@ const Footer = styled.div`
 
 const PaginationContainer = styled.div`
   padding: 0.875rem;
+  @media only screen and (max-width: 426px) {
```
We do not require this change

fragm3 commented on src/components/cms/SkillFeedbackPage/SkillFeedbackPage.js:
```diff
@@ -105,6 +113,10 @@ const EnabledButtonStyles = css`
   cursor: pointer;
 `;
 
+const AuthorName = styled.span`
```
Use either use class names or styled-components.

samagragupta replied:
Could not understand what do you mean?

samagragupta replied:
In this, I have used styled-components

fragm3 commented on src/components/cms/SkillFeedbackPage/SkillFeedbackPage.js:
```diff
@@ -51,6 +51,9 @@ const Container = styled(CenterReaderContainer)`
   width: 100%;
   font-size: 0.875rem;
+  @media only screen and (max-width: 426px) {
+    padding: 25px 12px;
```
Use rem instead.

fragm3 commented on src/components/cms/SkillFeedbackPage/SkillFeedbackPage.js:
```diff
@@ -590,12 +597,12 @@ class SkillFeedbackPage extends Component {
                   by{' '}
-                  <span
+                  <AuthorName
                     className="feedback-author"
```
Remove `className`

samagragupta replied:
@fragm3  fixed that

samagragupta commented:
@fragm3 Done

samagragupta commented:
@akshatnitd @harsh-jindal Please review
![feeeed](https://user-images.githubusercontent.com/32234926/60379572-09452400-9a53-11e9-83db-76807291909e.gif)
